### ISSUE TITLE
Cannot add albums to playlist

### ISSUE THREAD

**Issue by erezschatz** [ampache/ampache#3792]
### Describe the bug
The action of add to playlist ceased to work.

### To reproduce

1. Go to an album page
2. Click on 'Add to Playlist'
3. Select playlist
4. Notification pops up "added to playlist"
5. Go to playlist
6. Album not added.

### Expected behavior
Album should've been added to the playlist

## Environment

* Ampache version: 6.3.0 (530c9367a1150b731c7a0d81556eb91ada8c4e8c)
* Web server + version: Apache/2.4.52 (Ubuntu)
* Server operating system: Ubuntu 22.04.3 LTS
* Client operating system: MacOS/Linux (tested on Firefox on both)

### Client type

* Web

## Logs

```2024-01-25T07:00:46+00:00 [erez] (Ampache\Module\Application\ApplicationRunner) -> Found handler "Ampache\Module\Api\Edit\ShowEditPlaylistAction" for action "show_edit_playlist"
2024-01-25T07:00:46+00:00 [erez] (Ampache\Module\Api\Edit\AbstractEditAction) -> Called for action: {show_edit_playlist}
2024-01-25T07:00:46+00:00 [erez] (Ampache\Module\Api\Edit\AbstractEditAction) -> Ampache\Repository\Model\AlbumDisk
2024-01-25T07:00:46+00:00 [erez] (Ampache\Module\Api\Edit\AbstractEditAction) -> 32589
...
2024-01-25T07:01:00+00:00 [erez] (ajax.server) -> Called for page: {playlist}
2024-01-25T07:01:00+00:00 [erez] (playlist.ajax) -> Appending items to playlist {63}...
2024-01-25T07:01:00+00:00 [erez] (playlist.ajax) -> Adding all medias of album_disk(s) {32589}...
2024-01-25T07:01:00+00:00 [erez] (Ampache\Repository\Model\Playlist) -> add_medias to: 63
2024-01-25T07:01:00+00:00 [erez] (Ampache\Module\System\Dba) -> Error_query SQL: INSERT INTO `playlist_data` (`playlist`, `object_id`, `object_type`, `track`) VALUES (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?) [63,62639,"song",1,63,62640,"song",2,63,62642,"song",3,63,62637,"song",4,63,62636,"song",5,63,62633,"song",6,63,62641,"song",7,63,62634,"song",8,63,62635,"song",9,63,62638,"song",10,63,62632,"song",11]
2024-01-25T07:01:00+00:00 [erez] (Ampache\Module\System\Dba) -> Error_query MSG: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '63-1' for key 'playlist_track_UN'
2024-01-25T07:01:00+00:00 [erez] (Ampache\Module\System\Dba) -> Error_query SQL: INSERT INTO `playlist_data` (`playlist`, `object_id`, `object_type`, `track`) VALUES (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?) [63,62639,"song",1,63,62640,"song",2,63,62642,"song",3,63,62637,"song",4,63,62636,"song",5,63,62633,"song",6,63,62641,"song",7,63,62634,"song",8,63,62635,"song",9,63,62638,"song",10,63,62632,"song",11]
2024-01-25T07:01:00+00:00 [erez] (Ampache\Module\System\Dba) -> Error_query MSG: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '63-1' for key 'playlist_track_UN'
2024-01-25T07:01:00+00:00 [erez] (Ampache\Module\System\Dba) -> Error_query SQL: INSERT INTO `playlist_data` (`playlist`, `object_id`, `object_type`, `track`) VALUES (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?) [63,62639,"song",1,63,62640,"song",2,63,62642,"song",3,63,62637,"song",4,63,62636,"song",5,63,62633,"song",6,63,62641,"song",7,63,62634,"song",8,63,62635,"song",9,63,62638,"song",10,63,62632,"song",11]
2024-01-25T07:01:00+00:00 [erez] (Ampache\Module\System\Dba) -> Error_query MSG: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '63-1' for key 'playlist_track_UN'
2024-01-25T07:01:00+00:00 [erez] (Ampache\Module\System\Dba) -> Error_query SQL: INSERT INTO `playlist_data` (`playlist`, `object_id`, `object_type`, `track`) VALUES (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?), (?, ?, ?, ?) [63,62639,"song",1,63,62640,"song",2,63,62642,"song",3,63,62637,"song",4,63,62636,"song",5,63,62633,"song",6,63,62641,"song",7,63,62634,"song",8,63,62635,"song",9,63,62638,"song",10,63,62632,"song",11]
2024-01-25T07:01:00+00:00 [erez] (Ampache\Module\System\Dba) -> Error_query MSG: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '63-1' for key 'playlist_track_UN'
2024-01-25T07:01:00+00:00 [erez] (Ampache\Repository\Model\Playlist) -> Added 11 tracks to playlist: 63
2024-01-25T07:01:00+00:00 [erez] (playlist.ajax) -> Items added successfully!
2024-01-25T07:01:00+00:00 [erez] (Ampache\Module\System\Session) -> Writing to qd9v7ogdf2l3nnnn9868u6rb6g with expiration 1706169660```

**Comment by lachlan-00** [comment]
thanks i just put a constraint on the DB I'll fix now

**Comment by lachlan-00** [comment]
i wasn't getting the max track correctly.

fixed with f8811b820

**Comment by erezschatz** [comment]
Pulled from the repo, did a quick check, seems to be working. closing
